README.md

```diff
@@ -29,14 +38,18 @@ The example uses the [World Cities Dataset](https://simplemaps.com/data/world-ci
   }
 ```
 
-### Database Setup
+## Database setup
 
 It's recommended to use [`Postgres`] for your database so the schema can use `BigInt` and `Float` nicely.
 
 Also, the seed script uses Prisma's [`createMany()`](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#createmany) to bulk load records in batches which is significantly faster that loading data row by row -- which is important because we have ~43,000 world cities in the dataset.
 
 Unfortunately, `createMany` is not supported by SQLite.
```

Comment:
[sand] A minor addition to emphasize why Postgres is preferred. Based on the wording, I assume SQLite still works -- just not as well.
```suggestion
Unfortunately, `createMany` is not supported by SQLite. If you use a SQLite database, loading records will take significantly longer.
```